presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -510,6 +642,17 @@ private static WebApplicationException badRequest(String message)
                 .build());
     }
 
+    /**
+     * Check and return a given string if it is non-null and non-empty.
+     * <p>
+     * Returns the given string if is non-null and non-empty.
```

Comment:
I think we don't need this description.

Also, haven't you already documented a method similar this one in the PR before?